content/post/articles/Become-A-Software-Developer.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Become a Software Developer"
+date: 2021-11-30T15:32:19+01:00
+images: ["/Become-A-Software-Developer/shaba-rex.png"]
+categories: ["Software Developer"]
+tags: ["Software Developer","BootCamps","Self-taught Developer"]
+authors: ["Shaba-Rex"]
+---
+
+{{<figure src="/Become-A-Software-Developer/viva the it.jpeg"  alt="Software Engineering is Cool">}}
+
+Software engineering is the process of analyzing user needs and designing, constructing, and testing end-user applications that will satisfy these needs through the use of software programming languages. It is the application of engineering principles to software development.
+
+To be an Application Developer, good news there is not just one path, in this article we will talk about the different paths you have to take to become a software developer and the advantages and disadvantages of certain paths. But in the end it's up to you to decide which path is best for you.
+
+## Get an university Degree
+
+Going to college to graduate is the traditional path people take to become a developer.
+This path involves obtaining an associate's degree or a minimum bachelor's degree in computer science, computer programming, or a related field.But believe it or not going to college to be a software developer has its pros and cons.
+
+##### #Pros:
+
+it opens doors, indeed people with a degree in Computer science are more easily accepted by the technology industry, because the recruiter will look at the school where you did your classes and judge you according to the renowned of this school in terms of the courses offered by the school that graduated you.
+
+When you go to university to get a degree, you have several teachers who are much more present and attentive to your concerns, because you have a long period of time to perfect your education as a software developer.
+
+##### #Cons:
+
+Going to university takes a long time. Usually it takes at least three years to get a degree in computer science,which is too long for some people (like those who are in professional retraining or who are not interested in university courses because of their duration).
+
+Going to university is also very expensive. In Cameroon it is sometimes necessary to count at least 350,000 or 450,000 XAF just for a single year of university study in the Computer Science sector.
+
+The emphasis is much more on the theoretical aspect of the courses which sometimes displeases more than one.
+
+Do not forget one thing, the computer science degree is only a piece of paper, when you hire a developer, you hire him not for his beautiful eyes, but more for his skills, basically for what he is able to do.
+
+# Choose your BootCamp :
+
+{{<figure src="/Become-A-Software-Developer/Localhost academy.png"  alt="Image de la localHost Academy Douala">}}
```

Comment:
If you want to make a product placement, you probably want to discuss the financial matter with @elhmn. Your blog post now sounds more like an ad instead of an informative one.

Reply:
So let me explain what the issue is here. We are happy to see this particular bootcamp being mentioned on the article but we don't understand why this one in particular and not many other bootcamps. So we suspect that you tried to place this product in your article.

We won't be happy to merge this pull request if there is only one bootcamp mentioned in the article (we would like to have a list of at least 5 to feel alright with it). Or another solution is that you remove the mention of that bootcamp from the article.

Just note that if that bootcamp stays in the PR even with 4 other bootcamps, this PR won't provide you any Yotas as it would have normally been the case.

Reply:
Ok i got it. :)